nuget/release.sh

```diff
@@ -51,7 +51,7 @@ esac
 
 # Generate release notes for NuGet
 RELNOTES="$BUILDEXTRA"/nuget/ReleaseNotes.html
-RELNOTESMD="$BUILDEXTRA"/installer/ReleaseNotes.md
+RELNOTESMD="$BUILDEXTRA"/ReleaseNotes.md
```

Comment:
There are more mentions of `ReleaseNotes.md`... `please.sh` in particular needs to be made aware of the move, too.